src/main/java/ui/listpanel/ListPanelCell.java

```diff
@@ -54,6 +52,18 @@ public void updateItem(TurboIssue issue, boolean empty) {
         this.setId(issue.getRepoId() + "_col" + parentPanelIndex + "_" + issue.getId());
     }
 
+    private void updateStyleForClosedIssues(TurboIssue issue){
```

Comment:
Alternative logic (if open issues do not require styling):
if open but styled as closed, remove closed style
else if closed but styled as open, add closed style.